### PR TITLE
#164139659 Add search articles functionality

### PR DESCRIPTION
**Pull request by youngestdj**
#### What does this PR do?
This PR adds the feature for users to be able to search for articles
#### Description of tasks to be completed?
* Add feature for users to search articles
#### Any background context you would like to provide?
N/A
#### How can this be manually tested?
Go to the landing page and enter a search term in the search box
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/164139659

### CODE REVIEW

**Review by codinger41 — DISMISSED** [review]

**Review by Dubby20 — DISMISSED** [review]

**Review by olajide1234 — DISMISSED** [review]

File: src/views/landingPage/Index.jsx [hunk 1]
```diff
@@ -81,13 +146,31 @@ const LandingPage = (props) => {
...
-                variant="outline-secondary"
-                size="lg" block>
-                <h4>See more trending articles</h4>
-              </Button>
+              {
+                searchResults.searchRequest
+                && (searchResults.articlesCount > 6)
+                && ((searchResults.articles.length % 6) === 0) ? (
+                  <Button
+                    onClick={() => getMoreSearchResults()} // eslint-disable-line react/jsx-no-bind
```
Comment (9jaswag) [inline src/views/landingPage/Index.jsx]: @youngestdj I made mention of avoiding these inline ESLint disabling comments. If for any reason we're gonna do this, let's do it in the ESLint config

File: src/views/landingPage/Index.jsx [hunk 2]
```diff
@@ -81,13 +146,31 @@ const LandingPage = (props) => {
...
+                    variant="outline-secondary"
+                    size="lg" block>
+                    <h4>See more search results</h4>
+                  </Button>
+                  ) : null
+              }
+              {
+                !searchResults.searchRequest ? (
+                  <Button
+                    onClick={() => getMoreTrending()} // eslint-disable-line react/jsx-no-bind
```
Comment (9jaswag) [inline src/views/landingPage/Index.jsx]: @youngestdj remove this and other instances all through the codebase

**Review by 9jaswag — CHANGES_REQUESTED** [review]
quick fixes and this is merge ready

File: src/views/landingPage/Index.jsx (hunk 1 above)
Comment (youngestdj) [reply src/views/landingPage/Index.jsx]: Fixed

File: src/views/landingPage/Index.jsx (hunk 2 above)
Comment (youngestdj) [reply src/views/landingPage/Index.jsx]: Fixed

**Review by 9jaswag — APPROVED** [review]

**Comment by 9jaswag** [comment]
@Dubby20 @youngestdj please test this in develop and let me know

**Comment by youngestdj** [comment]
@9jaswag I have tested it in develop and it works.

**Comment by Dubby20** [comment]
The same here. It works on develop.

**Pull request merged**